packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -53,6 +60,12 @@ export const COMPOSITE_ONLY_ICONS = {
   /* @conditional-compile-remove-from(stable) Chat_Notification_Icon*/
   ControlBarButtonBadgeIcon: <Circle20Filled />,
   /* @conditional-compile-remove-from(stable): meeting/calling-composite */
+  ControlBarChatButtonActive: <Chat20Filled />,
+  /* @conditional-compile-remove-from(stable): meeting/calling-composite */
```

Comment:
change these conditional compiles to just `meeting-composite` and not `meeting/calling-composite` -- hopefully we'll have something soon to enforce this but having variations makes it hard to find-and-remove when stablizing the feature

Reply:
kk will do!